lib/octokit/client/service_status.rb

```diff
@@ -1,16 +1,14 @@
 module Octokit
   class Client
-
     # Methods for the GitHub Status API
     #
     # @see https://status.github.com/api
     module ServiceStatus
-
       # Root for status API
       # @private
-      SUMMARY_ROOT    = 'https://www.githubstatus.com/api/v2/summary.json'
-      STATUS_ROOT     = 'https://www.githubstatus.com/api/v2/status.json'
-      COMPONENTS_ROOT = 'https://www.githubstatus.com/api/v2/components.json'
+      SUMMARY_ROOT    = 'https://www.githubstatus.com/api/v2/summary.json'.freeze
```

Comment:
We might as well add the frozen string literal declaration to each of our files, rather than do this. We test with frozen strings enabled already, so it shouldn't matter.

Reply:
Issue created: https://github.com/octokit/octokit.rb/issues/1435